src/sweeps/params.py

```diff
@@ -325,24 +325,24 @@ def __init__(self, items: List[HyperParameter]):
         Args:
             items: A list of HyperParameters to construct the set from.
         """
+        self.searchable_params: List[HyperParameter] = []
+        self.param_names_to_index: Dict[str, int] = dict()
+        self.param_names_to_param: Dict[str, HyperParameter] = dict()
 
+        _searchable_param_counter: int = 0
```

Comment:
Can't use enumerate because constants are skipped and then indices are off by the number of constants
```
            normalized_runs[bayes_opt_index, non_nan_indices] = _param.cdf(row)[
>               non_nan_indices
            ]
E           IndexError: index 8 is out of bounds for axis 0 with size 7
```

Reply:
oop nice good call